src/main/java/be/syntra/devshop/DevshopBack/models/ProductDto.java

```diff
@@ -23,6 +23,7 @@
     private List<String> categoryNames;
     private Set<StarRating> ratings;
     private Double averageRating;
+    private int totalInCart;
```

Comment:
keep to the convention of not using the primitives please (except for booleans)

Reply:
removed this line, this property wasn't supposed to still be there